docs/Collaboration guidelines.md

```diff
@@ -0,0 +1,51 @@
+Collaboration Guidelines
+===
+
+Be polite to each other
+---
+
+No one wants to wake up to a rude reply. Try to behave the best way you can, and show understanding when others are not. 
```

Comment:
This comes off a bit pessimistic. Even if the tone of this specific point can't be changed, it could be moved after the next point so it starts off positive.